media/js/mozorg/manifesto.js

```diff
@@ -57,16 +57,30 @@ $(function() {
                         .text(window.trans('principle-nav-prev')).appendTo($nav);
 
                     $nav.on('click', 'button', function () {
-                        nav_modal($(this).hasClass('prev') ? -1 : 1);
-                    });
-
-                    window.dataLayer.push({
-                        event: 'manifesto-interaction',
-                        browserAction: 'modal open',
-                        section: section_id.match(/\d+/)[0]
+                        var $this = $(this);
+                        var quoteText = $('.principle.overlay-contents h3').text();
+                        // Track arrow clicks on Manifesto quote popup
+
+                        window.dataLayer.push({
+                            'event': 'manifesto-quote-click-arrow',
+                            'direction': ($this.attr('class') === 'prev') ? 'previous' : 'next',
+                            'quote': quoteText.slice(3),
```

Comment:
Yeah, I suggest hard coding GA specific values in `*-data` attributes in the markup. (We do this in [some places](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/desktop/customize.html#L46) already.)

For example, you could add `data-ga-quote-number` and `data-ga-quote` attributes to each `section.principle` in [manifesto.html](https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/templates/mozorg/about/manifesto.html#L92). Then, instead of parsing the text of each quote, you could pull the pre-defined `*-data` attributes. (These `*-data` attributes would not be wrapped for L10n, as they would only be used for GA.)

The above strategy may also help keep GA tidy, as you (along with Gareth?) could define short names for each quote instead of using the full quote text, e.g. "enrich-lives" instead of "The Internet must enrich the lives of individual human beings".